src/db/Place/place.ts

```diff
@@ -24,6 +24,14 @@ export async function getPlaceByID(id: GooglePlaceID["place_id"]): Promise<Place
 	return placeReturned;
 }
 
+export async function isPlaceInDb(id: GooglePlaceID["place_id"]): Promise<boolean> {
+	const placesCollection: Collection<Place> = await place;
```

Comment:
i know this is sorta random at this point but would you mind renaming `place` to `placeColPromise` or something like that and same for rating? For a sec I was like "what does await place mean? did we already fetch a place somewhere?"

Reply:
> 
> 
> i know this is sorta random at this point but would you mind renaming `place` to `placeColPromise` or something like that and same for rating? For a sec I was like "what does await place mean? did we already fetch a place somewhere?"

yeah that makes sense

Reply:
I like this, will do this right now!